### PR TITLE
Update inference sample links, Add sample inference inputs and outputs

### PR DESCRIPTION
**Pull request by susridhar**

### CODE REVIEW

File: models/hugging_face/openai-whisper-large/description.md [hunk 1]
```diff
@@ -6,5 +6,30 @@ Whisper is an OpenAI pre-trained speech recognition model with potential applica
...
 Batch | todo
+
+
+### Sample inputs and outputs (for real-time inference)
+
+#### Sample input
+```json
+{
+    "inputs": {
+        "audio": ["https://audiovisionfiles.blob.core.windows.net/audio/audio.m4a", "https://audiovisionfiles.blob.core.windows.net/audio/audio.m4a"],
```
Comment (praveenmathamsetty) [inline models/hugging_face/openai-whisper-large/description.md:19]: Change the 2nd audio file or keep single entry.

**Review by praveenmathamsetty — DISMISSED** [review]

File: models/hugging_face/openai-whisper-large/description.md (hunk 1 above)
Comment (susridhar) [reply models/hugging_face/openai-whisper-large/description.md:19]: We're testing two different languages

File: common/environments/python-sdk-v2/environment.yaml [hunk 2]
```diff
@@ -9,4 +9,4 @@ image:
     - conda_dependencies.yaml
   publish:
     location: mcr
-    visibility: unlisted
+    visibility: public
```
Comment (skasturi) [inline common/environments/python-sdk-v2/environment.yaml:12]: Merge issue I believe?

File: training/model_evaluation/environment/context/conda.yaml [hunk 3]
```diff
@@ -1,14 +1,15 @@
 name: model-evaluation
 channels:
```
Comment (skasturi) [inline training/model_evaluation/environment/context/conda.yaml:2]: Merge issue?

File: common/environments/python-sdk-v2/environment.yaml (hunk 2 above)
Comment (susridhar) [reply common/environments/python-sdk-v2/environment.yaml:12]: yes, fixing now

File: training/model_evaluation/environment/context/conda.yaml (hunk 3 above)
Comment (susridhar) [reply training/model_evaluation/environment/context/conda.yaml:2]: fixed

**Review by ManojBableshwar — APPROVED** [review]

**Review by skasturi — APPROVED** [review]

**Review by praveenmathamsetty — APPROVED** [review]

**Pull request merged**